src/main/resources/portal.properties.EXAMPLE

```diff
@@ -157,6 +157,10 @@ segfile.url=http://cbio.mskcc.org/cancergenomics/gdac-portal/seg/
 # Enable OncoKB annotation (true, false)
 show.oncokb=true
 
+# OncoKB API Token, for more information please check https://beta.oncokb.org/dataAccess
+# The token is required if you want to use OncoKB service
+oncokb.token=
+
```

Comment:
shouldn't the oncokb public URL property oncokb.public_api.url be here too? (and I would add a comment that if not provided, it default to the value from global properties)

Reply:
@sheridancbio it's never our intention to let other instances to specify oncokb link. This is just a property for our usage, so it's not documented

Reply:
@zhx828 so people can't deploy their own instance of OncoKB anymore? This might be a limitation for Dana Farber. They have some policy to not allow external API calls with mutation data

Reply:
@inodb technically, they still can. I will just write a document on how to do that.